[2023-09-21T07:37:13.108+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.install_dbt_deps manual__2023-09-21T07:34:11.085900+00:00 [queued]>
[2023-09-21T07:37:13.123+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.install_dbt_deps manual__2023-09-21T07:34:11.085900+00:00 [queued]>
[2023-09-21T07:37:13.123+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-21T07:37:13.328+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): install_dbt_deps> on 2023-09-21 07:34:11.085900+00:00
[2023-09-21T07:37:13.338+0000] {standard_task_runner.py:57} INFO - Started process 82 to run task
[2023-09-21T07:37:13.343+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'install_dbt_deps', 'manual__2023-09-21T07:34:11.085900+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpkvqd9v4k']
[2023-09-21T07:37:13.347+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask install_dbt_deps
[2023-09-21T07:37:13.436+0000] {task_command.py:415} INFO - Running <TaskInstance: main_dag.install_dbt_deps manual__2023-09-21T07:34:11.085900+00:00 [running]> on host 3e956870ddeb
[2023-09-21T07:37:13.568+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='install_dbt_deps' AIRFLOW_CTX_EXECUTION_DATE='2023-09-21T07:34:11.085900+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-21T07:34:11.085900+00:00'
[2023-09-21T07:37:13.571+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-21T07:37:13.572+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'dbt deps --profiles-dir /.dbt --project-dir /dbt_project']
[2023-09-21T07:37:13.589+0000] {subprocess.py:86} INFO - Output:
[2023-09-21T07:37:19.991+0000] {subprocess.py:93} INFO - [0m07:37:19  Running with dbt=1.5.3
[2023-09-21T07:37:19.993+0000] {subprocess.py:93} INFO - [0m07:37:19  Encountered an error:
[2023-09-21T07:37:19.994+0000] {subprocess.py:93} INFO - Runtime Error
[2023-09-21T07:37:19.994+0000] {subprocess.py:93} INFO -   No dbt_project.yml found at expected path /dbt_project/dbt_project.yml
[2023-09-21T07:37:19.994+0000] {subprocess.py:93} INFO -   Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
[2023-09-21T07:37:19.995+0000] {subprocess.py:93} INFO - 
[2023-09-21T07:37:22.954+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-09-21T07:37:22.972+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-09-21T07:37:22.976+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=main_dag, task_id=install_dbt_deps, execution_date=20230921T073411, start_date=20230921T073713, end_date=20230921T073722
[2023-09-21T07:37:23.015+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 26 for task install_dbt_deps (Bash command failed. The command returned a non-zero exit code 2.; 82)
[2023-09-21T07:37:23.043+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-09-21T07:37:23.085+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
